code/modules/client/loadout/loadout_accessories.dm

```diff
@@ -60,6 +60,42 @@
 	path = /obj/item/clothing/accessory/bonearmlet
 	slot = null
 
+//Factional
```

Comment:
No faction items in loadout.

Reply:
Thought someone approved it. Fair enough, I guess.
Though, I think solcon scarf is less factional and more patriotic. Think it'd fit well, overall.